crates/examples/src/bin/elfcopy.rs

```diff
@@ -205,7 +294,7 @@ fn copy_file<Elf: FileHeader<Endian = Endianness>>(
                 let s = in_dynamic_strings
                     .get(val.try_into()?)
                     .map_err(|_| "Invalid dynamic string")?;
-                Some(writer.add_dynamic_string(s))
+                Some(writer.add_dynamic_string(redefine_symbol_name(s)?))
```

Comment:
Are you sure you want to treat this as a symbol name? I don't think it is. Same for `verdaux.name`, `verneed.file` and `vernaux.name`.

Reply:
Removed